README.md

```diff
@@ -19,6 +19,8 @@ Every argument is optional.
 | `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages defined in the Cargo.toml file are processed. | |
 | `manifest-path`      | Path to Cargo.toml of crate or workspace to check. If not specified, the action assumes the manifest is under the default [`GITHUB_WORKSPACE`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) path. | |
 | `verbose`            | Enables verbose output of `cargo-semver-checks`. | `false` |
+| `rust-toolchain`     | Rust toolchain name to use, e.g. `stable`, `nightly` or `1.32.0`. | `stable` |
+| `rust-override`      | If Rust is already installed, whether to override it with version specified in `rust-toolchain`. | `false` |
```

Comment:
I think the override option in the `actions-rs/toolchain` action was a mistake, and I've already used it once to shoot myself in the foot. Let's avoid this option and always use what `rust-toolchain` is set to.

Reply:
Here's a possible design tweak for you to consider, and resolves the override problem while being very clear about edge cases.

This is the existing set of ways to override which Rust is used in an environment: https://rust-lang.github.io/rustup/overrides.html

The above means that if a project has a `rust-toolchain.toml` file, that will take precedence over any other configuration. (This is what the override was attempting to work around.)

Instead of fighting these other options which have precedence, let's embrace them:
- If the user's project or use of the action sets one of those overrides at the link, it is *the user's* responsibility to ensure an appropriate Rust is installed. This can be via our action's `rust-toolchain` option, or otherwise (e.g. if they've already installed it in an earlier step, before our action).
- `rust-toolchain` can be set to `""` (empty string) to _skip_ installing Rust altogether, and assume that an appropriate Rust has already been installed. This allows a user who is explicitly managing the Rust version to not end up installing Rust twice in the same workflow.

Reply:
This is a great idea! I pushed the changes, please have a look and check if I've understood you correctly :)